README.md

```diff
@@ -43,6 +43,14 @@ Besides functionality to perform a typical recommend-measure loop, BayBE's highl
 
 ## ⚡ Quick Start
 
+> [!IMPORTANT]

```

Comment:
on the necessary-sale I would only rate this a 2 out of 10, why is this needed?

Reply:
I just though I'll quickly add it because I stumbled across this possibility on another Github page I was reading. But the two of you don't really seem convinced... 🤔  If you prefer NOT to solve the issue that people run into this problems over and over again and keep asking us the same question, we can also drop this PR – I am honestly completely fine with it (and also, we now have a draft of how such a message can be created in our markdown file that we can reactivate later).

However, I must also say that I will then leave it to you to write the answers in case more people get lost, as I think this is a situation that can be completely avoided by a form of warning like the one suggested 🙃

Reply:
So I'll close the PR for now. If you come to the conclusion that we **should** add it, just answer here and we can reactivate it 👍🏼

Reply:
My main point is that I do not think that this solution will solve the issue: Since only the main page will be in the different version, can you elaborate on how this helps people that need other parts of the documentation? Or is this intended to only make them aware of the potential issue?

Reply:
Yeah, my motivation was to simply make the people aware that what they read here might not be compatible with the code they have installed

Reply:
Let's discuss if we want to have such a warning resp. in what way it might make sense to include it when we discuss about state of MultiDocs.

Reply:
ok, we can continue the discussion in the multidocs PR then, once it becomes relevant